x-pack/packages/ml/trained_models_utils/src/constants/trained_models.ts

```diff
@@ -7,6 +7,18 @@
 
 import { i18n } from '@kbn/i18n';
 
+export const ELSER_MODEL_ID = '.elser_model_2';
+export const ELSER_LINUX_OPTIMIZED_MODEL_ID = '.elser_model_2_linux-x86_64';
+export const E5_MODEL_ID = '.multilingual-e5-small';
+export const E5_LINUX_OPTIMIZED_MODEL_ID = '.multilingual-e5-small_linux-x86_64';
+export const LANG_IDENT_MODEL_ID = 'lang_ident_model_1';
+export const ELSER_ID_V1 = '.elser_model_1' as const;
```

Comment:
shall we use this `const` as keys for `ELASTIC_MODEL_DEFINITIONS`?

Reply:
Done in https://github.com/elastic/kibana/pull/187154/commits/9df4b804b06244bb2f9f4cacb338a8a0abe7c931